pyntc/devices/base_netmiko.py

```diff
@@ -0,0 +1,50 @@
+"""Module for netmiko base driver."""
+from typing import Any  # , List, Union, Iterable, Callable, Mapping
+
+from .base_device import BaseDevice, fix_docs
+
+# from pyntc.errors import CommandError, CommandListError
+
+
+@fix_docs
+class BaseNetmikoDevice(BaseDevice):
+    """Netmiko Base Driver."""
+
+    def __init__(
+        self,
+        host: str,
+        username: str,
+        password: str,
+        device_type: str,
+        secret: str = "",
+        port: int = 22,
+        **netmiko_args: Any,
```

Comment:
Why not include `global_delay_factor` as a property? It seems that it is used by ConnectHandler. It can be part of `**netmiko_args`, I just find it more clear when a property is common to use the specific property.

Reply:
I would prefer not to define netmiko's optional or keyword args as changes in netmiko's calling signature do not affect us. For example, if Netmiko deprecated `global_delay_factor`, we would have to adjust our calling signature depending on what version of Netmiko the user had installed.

Reply:
Makes sense, @jmcgill298. Thanks for the response!